.golangci.yml

```diff
@@ -18,6 +18,9 @@ linters:
     - unconvert
     - unused
     - varcheck
+    - stylecheck
```

Comment:
I'm not sure if `stylecheck` is needed, we have a better alternative already that is `revive`, perhaps you should check `revive` docs and enable the necessary linting rule that will do exactly what we're looking for?

Reply:
I checked the `revive` linter; it is a complex one compared to the `stylecheck`. It is a standalone linter competing with `golanlig lint`, and I do not think we need to use it for such simple checks. Also, I cannot find any k8s or terraform projects using it.